build_tools/benchmarks/trigger_db_import.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+# Copyright 2023 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# Triggers the cloud function which imports the benchmark results from the given
+# GitHub Actions run into the database.
+#
+# Usage: trigger_db_import.sh <importer> <github_run_id> <github_run_attempt>
+#
+# - <importer> is either 'presubmit' or 'postsubmit'
+# - <github_run_id> is an integer identifier
+# - <github_run_attempt> is an integer identifier
+#
+# /<github_run_id>/<github_run_attempt>/ is the path in the bucket where the cloud function is
+# looking for the benchmark results. The bucket is determined by the <importer> option.
+# Note, that you can't trigger an import from an arbitrary bucket for security reasons.
+
+set -euo pipefail
+
+readonly SCRIPT_DIR="$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)"
```

Comment:
It looks like we are using a shorter version like this one https://github.com/openxla/iree/blob/134f36e7f52f78eded3dd260000ee472283bbf86/build_tools/github_actions/runner/config/setup.sh#L14

Maybe we can use the same command here?

Reply:
Yes. Changed. Thx.